sysroot/Files/Tests/xml/basic.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<Root TestAttr="0001" TestType="CMD">
+  <Name>Convert number to string</Name>
+  <CommandLine>Examp1.EXE</CommandLine>
+  <Input>1</Input>
+  <Output>One</Output>
+</Root>
```

Comment:
We could have avoided the need for external files by using a C++ raw string ;)

```C++
R"xml(
  <?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
  <Root TestAttr="0001" TestType="CMD">
    <Name>Convert number to string</Name>
    <CommandLine>Examp1.EXE</CommandLine>
    <Input>1</Input>
    <Output>One</Output>
  </Root>
)xml"
```

https://en.cppreference.com/w/cpp/language/string_literal

Reply:
I used that before, but i was thinking about adding more & more files and also moving the test png files out. Having these as raw strings won't work since they're binary

Reply:
Ok ok